modules/apps/27-interchain-accounts/host/keeper/events.go

```diff
@@ -1,20 +1,29 @@
 package keeper
 
 import (
+	"fmt"
 	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/ibc-go/v3/modules/apps/transfer/types"
```

Comment:
I see this is imported from `transfer` for `AttributeKeyAckSuccess`. Could we redefine this in the `interchain-accounts` module package `types` so we don't have apps depending directly on one another unnecessarily?